_posts/2016-05-10-Developers-love-spaces.md

```diff
@@ -0,0 +1,54 @@
+
+![Would you indent code with tabs or spaces?](/img/devs-love-spaces/spaces-vs-tabs.png)
+
+Other trends are also visible in the survey: For maintaining configurations - an important part of doing devops, or doing things right - there are many tool choices available. Here in Solita, the preferred choice is Ansible - with all its pros and cons 42% of devs choose it for automating and documenting the configurations. Chef and Puppet together don't come even close. Many still use traditional shell scripting (21%), or have to do manual work (16%). Another way to look at these statistics is to say that 84% of developers work in happily automated configuration management world, while the rest 16% just have to suck it.
+
+![How do you manage?](/img/devs-love-spaces/configuration-management.png)
+
+Then there's other, more obvious statistics. When it comes to version control, Git is way cooler than anything else currently. Most popular tools are IntelliJ IDEA, and Eclipse, while there's still solid 2,4% Vim-coders out there. For rapid text edit work people often whip out Sublime or Notepad. Most devs are running OSX on their machines here, with almost equal amount running Windows. 12% are running some brand of Linux. 
+
+## How about frameworks and languages?
+
+Most popular languages being used here are: Java, Clojure, Javascript, and C#. Anything else is just peanuts when it comes to statistics. Interestingly, Clojure is the most preferred language for devs, so I suppose we just can't get enough of Clojure in daily work.
+
+For the frontend development, most of the work is done with Angular, with React + Reagent catching up. Most people would choose to use React for the next project - but then most people haven't yet been exposed to Angular 2 so we'll see about that in 2017 ;) 
+
+## Roles and experience
+
+Most of our coders have been in business over 11 years - so they've seen a lot. 95% have been coding at least 6 years - or 75% if we only count coding for bread and butter, not for fun.
+
+It seems most of people in dev role consider themselves as fullstack developers - pretty much half - with 10% specialized back-end developers, 10% architects, and 5% slices of front-end developers, and integration specialists.
+
+![Roles in projects?](/img/devs-love-spaces/roles.png)
+
+About 74% of Solita developers contribute some amount of work to open source projects, on their own free time. Naturally, some projects allow to donate open source code also on within the projects. 
+
+## Conclusion
+
+So: We do a lot with Java and Angular, and would currently love to do more with Clojure and React/Reagent. It also seems we're pretty modest - only 16% of devs believe their skills put them to the best quarter of coders - majority thinks they are in the second best quarters.
```

Comment:
Tännekin laittaisin kaksoispisteen jälkeen pienen alkukirjaimen. Mietin ylipäänsä, onko tuo So: -alku tarpeellinen.

only 16% of devs believe their skills put them to the best quarter of coders - ihan niin kuin puuttui joku sana?
believe _in_ their skills and(?) put themselves(?) in to the best quarter of coders... tai consider themselves as the best coders in Solita.

the majority

second best quarter, monikon s pois